Action rightArmAction
Action leftArmAction

Target rightArmTarget
Target leftArmTarget
---
bool success
